exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -1680,6 +1695,14 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
                             }
                         }
                         else -> checkConstraints
+                    }.let {
+                        if (currentDialect !is SQLiteDialect && currentDialect !is OracleDialect) {
+                            it.filterNot { (name, _) ->
+                                name.startsWith("${generatedSignedCheckPrefix}integer")
+                            }
```

Comment:
SQLite and Oracle now defining a check constraint on the column is also a breaking change, especially if user's were taking advantage of the underlying database type and using plain sql.
Please add a note to the breaking changes that the DDL has changed and `long()` should be used if they plan to go outside the range, or something similar to what's been done [before](https://jetbrains.github.io/Exposed/breaking-changes.html#0-55-0).